File: src/test/java/no/utgdev/diy/ex2/Ex2Config2.java
package no.utgdev.diy.ex2;

//REMOVE import no.utgdev.diy.annotations.Bean;

import java.util.Collections;
import java.util.Map;

public class Ex2Config2 {
    //REMOVE @Bean
    public double number() {
        return 2.0;
    }

    //REMOVE @Bean
    public Map<String, String> map() {
        return Collections.emptyMap();
    }
}

File: src/test/kotlin/no/utgdev/diy/ex2/Ex2Config2.kt
package no.utgdev.diy.ex2

import no.utgdev.diy.annotations.Bean;

class Ex2Config2 {
    @Bean
    fun number(): Double {
        return 2.0
    }

    @Bean
    fun map(): Map<String, String> {
        return emptyMap()
    }
}